Gopkg.lock

```diff
@@ -761,6 +761,7 @@
     "pkg/version",
     "plugin/pkg/client/auth/exec",
     "plugin/pkg/client/auth/gcp",
+    "plugin/pkg/client/auth/oidc",
```

Comment:
Is necessary for the issue being fixed?

Reply:
This is a generated file, but not sure why it is checked-in...